omegaconf/basecontainer.py

```diff
@@ -220,6 +220,7 @@ def convert(val: Node) -> Any:
             for key in conf.keys():
                 node = conf._get_node(key)
                 assert node is not None
+                assert isinstance(node, Node)
```

Comment:
```suggestion
                assert isinstance(node, Node)
```

No need for the first assertion, since the second one implies `node is not None`.

Reply:
roger.